back-end/sports_platform/sports_platform_api/validation/event_validation.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import serializers
+
+class Event(serializers.Serializer):
+
+    name = serializers.CharField(min_length=2, max_length=100, required=True)
+    sport = serializers.CharField(required=True)
+    description = models.CharField(required=False)
+
+    startDate = models.DateTimeField(format='iso-8601', required=True)
```

Comment:
I do not know if it does this but we should check that this should contain time and date together.

Reply:
I could not find a validator specific to this. But I think checking if the string contains T would work. What do you think?

Reply:
sure, it makes sense, thanks!